src/mgmt/svc.pb-c.c

```diff
@@ -937,13 +937,39 @@ static const ProtobufCFieldDescriptor mgmt__join_req__field_descriptors[9] =
     0,             /* flags */
     0,NULL,NULL    /* reserved1,reserved2, etc */
   },
+  {
+    "secondary_uris",
+    10,
+    PROTOBUF_C_LABEL_REPEATED,
+    PROTOBUF_C_TYPE_STRING,
+    offsetof(Mgmt__JoinReq, n_secondary_uris),
+    offsetof(Mgmt__JoinReq, secondary_uris),
+    NULL,
+    &protobuf_c_empty_string,
+    0,             /* flags */
+    0,NULL,NULL    /* reserved1,reserved2, etc */
```

Comment:
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)